src/cmdlib.sh

```diff
@@ -320,6 +320,23 @@ EOF
     fi
     if [[ -n $(ls "${overridesdir}/rpm/"*.rpm 2> /dev/null) ]]; then
         (cd "${overridesdir}"/rpm && rm -rf .repodata && createrepo_c .)
+        # synthesize an override lockfile to force rpm-ostree to pick up our
+        # override RPMS -- we try to be nice here and allow multiple versions of
+        # the same RPMs: the `dnf repoquery` below is to pick the latest one
+        local_overrides_lockfile="${tmp_overridesdir}/local-overrides.json"
+        dnf repoquery  --repofrompath=tmp,"file://${overridesdir}/rpm" \
```

Comment:
Heh.  I started to look at doing this in rpm-ostree, but shelling out to `dnf` is OK too I guess for now.